tutorials/13_diagnosis_sbc.ipynb

```diff
@@ -0,0 +1,1897 @@
+      "text/plain": [
+       "<Figure size 576x360 with 2 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "f, ax = sbc_rank_plot(ranks, 1_000, plot_type = \"hist\", num_bins=30)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "collapsed": false
+   },
+   "source": [
+    "We can see that the Kolmogorv-Smirnov p-values vanish, we can reject the hypothesis that the `ranks` PDF is uniform PDF. The c2st accuracies show values higher than `0.5`. This is indicative that something is wrong as it emphasizes again the `ranks` is not a uniform PDF. Let's try to shift all posterior samples in the opposite direction. Because we have shifted the expected value of the posterior to higher values, we see more ranks at low values."
```

Comment:
make this more explicit, e.g., plot the KS results directly above this cell or add a pointer to the cell above in parentheses or so.

Reply:
5d6fa01